include/deal.II/lac/scalapack.h

```diff
@@ -189,6 +189,163 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
                const std::pair<unsigned int,unsigned int> &offset_B,
                const std::pair<unsigned int,unsigned int> &submatrix_size) const;
 
+  /**
+   * Transposing assignment: <i>A = B<sup>T</sup></i>
+   *
+   * The matrices <tt>A</tt> and <tt>B</tt> must have the same process grid
+   *
+   * Following alignment conditions have to be fulfilled: MB_A = NB_B and NB_A = MB_B
+   */
+  void copy_transposed(const ScaLAPACKMatrix<NumberType> &B);
+
+  /**
+   * Matrix-addition:
+   *
+   * <i>A = a A + b op(B)</i>
+   *
+   * if(transpose_B)  <i>op(B) = B<sup>T</sup></i>
+   *
+   * else  <i>op(B) = B</i>
+   *
+   * The matrices <tt>A</tt> and <tt>B</tt> must have the same process grid
+   *
+   * Following alignment conditions have to be fulfilled:
+   *
+   * | transpose_B |          Block Sizes         |
+   * | :---------: | :--------------------------: |
+   * |   false     | MB_A = MB_B <br> NB_A = NB_B |
+   * |   true      | MB_A = NB_B <br> NB_A = MB_B |
+   */
+  void add(const ScaLAPACKMatrix<NumberType> &B,
```

Comment:
now i see what you wanted to implement, elsewhere (for vector classes) we call this:

```
sadd (const Number s, const Number a, const VectorSpaceVector< Number > & V)
```

since other matrix classes don't support this operation, i would rather stick with

```
sadd (const .. s, const ... a,const ... B, const bool transpose_B = false)
```

@dealii/dealii any opinions on naming of this matrix operation?